conf/modules.config

```diff
@@ -252,17 +252,15 @@ process {
     */
 
     if(params.annotate){
-        withName: "^.*:ANNOTATION:ENSEMBLVEP_VEP\$" {
+        withName: "^.*:VCF_ANNOTATE_ENSEMBLVEP:ENSEMBLVEP_VEP\$" {
             // errorStrategy = { task.exitStatus = 255 ? 'retry' : 'terminate' }
             // maxRetries = 3
             // container = {params.vep_cache ? "quay.io/biocontainers/ensembl-vep:${params.vep_version}--pl5321h4a94de4_${task.attempt - 1}" : "nfcore/vep:${params.vep_version}.${params.genome}"}
+            memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
             container = "docker.io/nfcore/vep:${params.vep_version}.${params.genome}"
-            ext.prefix = {"${meta.id}_${meta.regions.contains(",") ? "alt" : meta.regions }"}
             ext.args = {[
                 // Specify the input format
                 "--format vcf",
-                // select the chromosomes to run on
-                "--chr ${meta.regions}",
                 // don't contact external db
                 '--offline',
                 // increase buffer_size to speed up analysis
```

Comment:
should we set `--buffer-size` to `vep_chunk_size`?

Reply:
Good idea!!